eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/configuration/InputParametersConfiguration.java

```diff
@@ -28,4 +30,10 @@
     public InputParameters inputParameters() {
         return new InputParameters();
     }
+
+    @Bean
+    @ConfigurationProperties(prefix = "eva.count-stats")
+    public CountParameters countParameters() {
```

Comment:
I think this should have it's own class and not be in `InputParameters`

Reply:
updated as decided